paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentOptionsViewModel.kt

```diff
@@ -292,7 +291,6 @@ internal class PaymentOptionsViewModel @Inject constructor(
     }
 
     private fun processNewPaymentMethod(paymentSelection: PaymentSelection) {
-        prefsRepository.savePaymentSelection(paymentSelection)
```

Comment:
Don't think we should be saving the selected option in `PrefsRepository` until after choosing to pay with it.